### DISCUSSION TITLE
[Atcoder] ABC 348 E - Minimize Sum of Distances

### DISCUSSION THREAD

**Discussion by jojaeng2** [BDD-CLUB/challenge-every-week-posting#436] [1. 개발 관련 글 모음]
지난주에 앳코더에서 못푼 문제를 다시 풀면서 풀이를 적어봤고, rerooting이라는 알고리즘을 알게되어 정리도 해봤습니다.
[blog](https://blog.naver.com/ds4ouj/223413883053)

**Comment by 02ggang9** [comment]
잘 봤습니다~!

**Reply by jojaeng2** [reply]
감사합니다 ㅎ
